Ddaangn/src/main/java/com/dev/ddaangn/user/User.java

```diff
@@ -79,9 +81,9 @@ public String getRoleKey() {
 
     // 연관관계 매핑
     // User - Badge
-//    @OneToMany(mappedBy = "user")
-//    private List<Badge> badges = new ArrayList<>();
-//
+    @OneToMany(mappedBy = "user")
+    private List<Badge> badges = new ArrayList<>();
+
 //    // User - AvatarImage TODO, Image 상속받고 만들기
```

Comment:
```suggestion

public void addBadgest(Badge badge) {
  this.badges.add(badge);
}

public void removeBadgest(Badge badge) {
  this.badges.remove(badge);
}

```

이 작업이 필요할 ㄱ ㅓㅅ 가습니다.